app/src/main/java/com/example/dispatcher/presentation/auth/AuthViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.dispatcher.presentation.favorites.viewModel
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.example.dispatcher.presentation.auth.AuthManager
+import com.example.dispatcher.presentation.auth.AuthResult
+
+class AuthViewModel(application: Application) : AndroidViewModel(application) {
+    private val authManager = AuthManager()
+
+    private val stage = MutableLiveData<String>()
+    private val _authResult = MutableLiveData<AuthResult>()
+    val authResult: LiveData<AuthResult> get() = _authResult
```

Comment:
i don't understand why some of the live data are private and some are not. be consistent.